packages/amplify-console-integration-tests/src/pullAndInit/amplifyConsoleOperations.ts

```diff
@@ -26,7 +26,11 @@ export async function deleteAllAmplifyProjects(amplifyClient?: any) {
 
 export async function deleteAmplifyStack(stackName: string, cfnClient?: any) {
   if (!cfnClient) cfnClient = getConfiguredCFNClient();
-  await cfnClient.deleteStack({ StackName: stackName }).promise();
+  try {
+    await cfnClient.deleteStack({ StackName: stackName }).promise();
+  } catch (err) {
+    // do nothing
```

Comment:
nit: Elaborate the comment to explain why nothing happens here

Reply:
This is mainly added since the stack may not exist and it will through an error. Though for this kind of err we can ignore as it's not pertinent to the test.